LICENSE

```diff
@@ -198,4 +198,4 @@
    distributed under the License is distributed on an "AS IS" BASIS,
    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
    See the License for the specific language governing permissions and
-   limitations under the License.
+   limitations under the License.
```

Comment:
Just curious what changed here.

Reply:
There was an empty line at the end, it was the only way I could think of to trigger another CI build since I didn't have access on Travis to restart (in case it just failed of no reason).